includes/admin/helpers/class-notices.php

```diff
@@ -30,11 +30,6 @@ public static function show_bell() {
 			<a href="#" class="wpbdp-bell-notifications-close"><?php esc_html_e( 'Hide notifications', 'business-directory-plugin' ); ?></a>
 			<ul class="wpbdp-bell-notifications-list"></ul>
 		</div>
-		<div class="wpbdp-bell-notification">
-			<a class="wpbdp-bell-notification-icon" href="#">
```

Comment:
I'm seeing some leftover references to ".wpbdp-bell-notification" and "wpbdp-bell-notification-icon". Can we remove more of this?

Reply:
I'm fine with that. Thank you!

Reply:
@shervElmi I'm seeing css that isn't being used that would be nice to remove. It's much easier to do now than later.